db/schema.rb

```diff
@@ -77,25 +109,26 @@
     t.datetime "updated_at", null: false
     t.integer  "book_id"
     t.index ["book_id"], name: "index_ratings_on_book_id", using: :btree
+    t.index ["user_id", "book_id"], name: "index_ratings_on_user_id_and_book_id", unique: true, using: :btree
     t.index ["user_id"], name: "index_ratings_on_user_id", using: :btree
   end
 
   create_table "users", force: :cascade do |t|
     t.string   "name"
-    t.datetime "created_at",                                     null: false
-    t.datetime "updated_at",                                     null: false
-    t.string   "email",                             default: "", null: false
-    t.string   "encrypted_password",                default: "", null: false
+    t.datetime "created_at",                                        null: false
+    t.datetime "updated_at",                                        null: false
+    t.string   "email",                             default: "",    null: false
+    t.string   "encrypted_password",                default: "",    null: false
     t.string   "reset_password_token"
     t.datetime "reset_password_sent_at"
     t.datetime "remember_created_at"
-    t.integer  "sign_in_count",                     default: 0,  null: false
+    t.integer  "sign_in_count",                     default: 0,     null: false
     t.datetime "current_sign_in_at"
     t.datetime "last_sign_in_at"
     t.inet     "current_sign_in_ip"
     t.inet     "last_sign_in_ip"
     t.string   "authentication_token",   limit: 30
-    t.boolean  "admin"
+    t.boolean  "admin?",                            default: false
```

Comment:
this is wrong, the variable should be just admin, and then rails provides the class with a method admin?